app/app/templates/shared/hackathon-list.html

```diff
@@ -1,87 +1,287 @@
+                      {% if event.logo %}
+                        <img class="hackathon-card-logo" src="{{ MEDIA_URL }}{{ event.logo }}" alt="Hackathon logo" />
+                      {% else %}
+                        <div class="hackathon-card-logo text-center px-3 font-caption">
+                          {{ event.name }}
+                        </div>
+                      {% endif %}
+                      </a>
+                    </div>
+                    <div class="card-body col-12 d-flex flex-column justify-content-between" style="height: calc(100% - 240px);">
+                      <div class="title-and-dates">
+                        <h4 {% if force_color %}style="font-size: 1.5rem;"{% endif %}>
+                          <a href="{% url 'hackathon' event.slug %}" class="text-grey-500 line-height-2 {% if force_color %}no-pink{% endif %} text-left line-clamp-2" rel="noopener noreferrer">
+                            {{ event.name }}
+                          </a>
+                        </h4>
 
-          <div class="card-header text-center col-12 col-sm-4 px-0">
-            <a href="{% url 'hackathon' event.slug %}">
-            {% if event.logo %}
-              <img class="hackathon-card-logo" src="{{ MEDIA_URL }}{{ event.logo }}" alt="Hackathon logo" />
-            {% else %}
-              <div class="hackathon-card-logo text-center px-3 font-caption">
-                {{ event.name }}
-              </div>
-            {% endif %}
-            </a>
+                        <div class="font-smaller-1 text-primary {% if left_align_cards %}text-left{% endif %}">
+                          <time class="font-weight-bold" datetime="{{ event.start_date|date_fromisoformat:'c' }}">{{ event.start_date|date_fromisoformat:"M d" }}</time>
+                          <span>-</span>
+                          <time class="font-weight-bold" datetime="{{ event.end_date|date_fromisoformat:'c' }}">{{ event.end_date|date_fromisoformat:"M d Y" }}</time>
+                          {% if event.total_prize %}<b><span class="px-2">•</span> {{ event.total_prize }}</b>{% endif %}
+                        </div>
+                      </div>
+
+                      <div class="mt-3 font-smaller-1 line-height-2 text-grey-400 hackathon-summary {% if left_align_cards %}text-left{% endif %}" style="min-height:80px;">
+                        <p>{{ event.summary }}</p>
+                      </div>
+
+                      <div class="mt-auto text-left">
+                        {% if event.sponsor_profiles|length %}
+                          <div class="mt-3 font-smaller-2">
+                            <div class="mb-1 text-grey-400">Sponsored by</div>
+                            {% for sponsor in event.sponsor_profiles|slice:":6" %}
+                              <a href="{{ sponsor.absolute_url }}" target="_blank"><img class="rounded-circle my-1 mr-1" style="width:36px;" src="{{ sponsor.avatar_url }}" /></a>
```

Comment:
worth adding `noopener/noreferrer` ?

Reply:
We could do, but these are internal links - they're pointing to the sponsors profile (this code is unchanged from the original `hackathon-list.html` page)